be/src/Project/Web/Seller/Seller.Web/Areas/Products/Repositories/CategoriesRepository.cs

```diff
@@ -222,7 +222,7 @@ public async Task<CategorySchema> GetCategorySchemaAsync(string token, string la
                 return response.Data;
             }
 
-            if (!response.IsSuccessStatusCode && response.StatusCode == HttpStatusCode.NotFound)
+            if (!response.IsSuccessStatusCode && response.StatusCode == HttpStatusCode.NoContent)
```

Comment:
**suggestion:** Hi @SebaKas21 , this requires adjustment because the 204 is considered as a success code in contrast to the not found. Please check other places if there isn't anything like that. It should be simply:

if (response.StatusCode == HttpStatusCode.NoContent)